.github/workflows/ScriptTests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
         with:
           global_files: |
-            !{packages/**/*,scripts/**/*}
+            packages/**/*
```

Comment:
The name of this variable makes no sense now. And shouldn't most of these checks be for a specific package, rather than both? As far as I can tell, only the `npm run test:scripts` command depends on both packages.

Reply:
Changes to `generator-widget` should also trigger ScriptTests because it uses the latest unpublished generator-widget package to generate the "latest" cases and then run tests against it

Agree with the name change, what name should it be ?

Reply:
After the latest changes, `packages_files` would make sense. And the setup with `npm install` is _much_ simpler in the worfklow now 👍🏻.

Reply:
Nice! `global_files` replaced with `packages_files`

Reply:
✨